apps/src/code-studio/pd/fit_weekend_registration/TravelPlans.jsx

```diff
@@ -135,9 +131,9 @@ export default class TravelPlans extends LabeledFormComponent {
           }
           {
             this.props.data.needHotel === 'Yes' &&
-            this.props.data.needAda === 'Yes' &&
-            this.largeInputFor("explainAda")
+            this.props.data.needAda === 'Yes'
```

Comment:
Did you mean to remove the large input for explaining ADA requirements? Right now, lines 133 and 134 just render something or nothing

Reply:
Sorry - I meant to say 133 and 134 can be removed. They do nothing

Reply:
That's right, I will remove those lines